core/new-gui/src/app/dashboard/component/feature-container/user-project-list/user-project-section/ngbd-modal-add-project-workflow/ngbd-modal-add-project-workflow.component.ts

```diff
@@ -13,12 +14,12 @@ import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
   styleUrls: ["./ngbd-modal-add-project-workflow.component.scss"],
 })
 export class NgbdModalAddProjectWorkflowComponent implements OnInit {
-  @Input() addedWorkflows!: DashboardWorkflowEntry[];
   @Input() projectId!: number;
 
   public unaddedWorkflows: DashboardWorkflowEntry[] = [];
   public checkedWorkflows: boolean[] = [];
   private addedWorkflowKeys: Set<number> = new Set<number>();
+  private addedWorkflows: DashboardWorkflowEntry[] = [];
```

Comment:
Can you add some comments here to explain the meaning of each variable?

Reply:
Completed, added comments explaining here